distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -76,6 +77,17 @@ type Type
         c = self.value
         get_short_type_name c
 
+    ## ADVANCED
+       GROUP Metadata
+
+
+    find : Text -> Type ! Not_Found
+    find qualified_name =
```

Comment:
A simple method to search **all loaded modules** for a type by its FQN. Trying to search in the scope of the caller would be problematic - there is not enough information in the runtime.